views/beneficial-owner-type.html

```diff
@@ -54,6 +54,9 @@
 or (managing_officers_corporate and managing_officers_corporate.length > 0) %}
 
 {% block pageTitle %}
+  {% if errors and errors.errorList and errors.errorList.length > 0 %}
```

Comment:
could add a test to make sure the 'Error:' appears in the title when there are errors